### PR TITLE
CSS styling

### PR DESCRIPTION
**Pull request by millipede-cpu**
Hi Orian, I've added to the CSS styling, with a background image that fills the page and more emojis for icons. I checked the Lighthouse Audit for accessibility, and changed the contrast of some colour ratios, and added aria labels 👍🏼

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -16,11 +16,11 @@ <h1>To Do List</h1>
     <form action="" method="POST">
         <input name="task" id="task" type="text" placeholder="Add a task" required/>
 
-        <input id="add-btn" type="submit" value="+" required/>
+        <button id="add-btn" type="submit" aria-label="Add"><i class="far fa-plus-square"></i></button>
     </form>
 
     <div class="task-box">
-    <button id="filter-btn">Hide completed tasks</button>
+    <button id="filter-btn" aria-label="Add button">Hide completed tasks &#x1F648</button>
```
Comment (OrianP) [inline index.html]: Maybe the aria-label should be "hide completed tasks" to describe the button more accurately?

File: index.html [hunk 2]
```diff
@@ -7,7 +7,7 @@
     <link rel="stylesheet" href="styles.css">
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link href="https://fonts.googleapis.com/css2?family=Gelasio:ital,wght@0,400;0,500;1,600&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Fredericka+the+Great&display=swap" rel="stylesheet">
```
Comment (OrianP) [inline index.html]: I personally prefer Gelasio font. From a user experience, I think the font should be simple and very easy to read and so I think Federicka The Great is very pretty but a little too decorative for the task list input. What do you reckon?

**Review by OrianP — CHANGES_REQUESTED** [review]
Hi Milly,🌷 
Great work! I just added two little comments on the index.html. Let me know your thoughts.

File: index.html (hunk 1 above)
Comment (millipede-cpu) [reply index.html]: Oops yes I just realised I was labelling the wrong button, I've updated this to "hide completed tasks"

File: index.html (hunk 2 above)
Comment (millipede-cpu) [reply index.html]: I think you're right, I was trying to find a typewriter font to match the background, but on closer inspection, it is hard to read sometimes. I've swapped the font back to Gelasio.

**Review by OrianP — APPROVED** [review]
Great, thank you for making the changes 🙏

**Pull request merged**